server/routes/hearingOutcome/damagesOwed/damagesOwedValidation.ts

```diff
@@ -0,0 +1,38 @@
+import { FormError } from '../../../@types/template'
+
+type DamagesOwedForm = {
+  damagesOwed: string
+  amount: string
+}
+
+const errors: { [key: string]: FormError } = {
+  MISSING_DAMAGES_OWED_CHOICE: {
+    href: '#damagesOwed',
+    text: 'Select yes, if any money is being recovered for damages.',
```

Comment:
In Anne's spreadsheet she has this as "Select yes if any money is being recovered for damages"
so i think we need to remove the comma after yes, and remove the fullstop